v-next/hardhat/src/internal/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const HARDHAT_NAME = "Hardhat";
```

Comment:
This needs to be merged with `v-next/hardhat/src/internal/cli/init/constants.ts`